modules/recommended-etcd-practices.adoc

```diff
@@ -64,7 +72,7 @@ $ sudo docker run --volume /var/lib/etcd:/var/lib/etcd:Z quay.io/openshift-scale
 ----
 --
 
-The output reports whether the disk is fast enough to host etcd by comparing the 99th percentile of the fsync metric captured from the run to see if it is less than 10 ms.
+The output reports whether the disk is fast enough to host etcd by comparing the 99th percentile of the fsync metric captured from the run to see if it is less than 20 ms.
```

Comment:
On this page, there is one instance where fsync is in backticks: 

Slow disks and disk activity from other processes can cause long `fsync` latencies.

Not sure whether it should/shouldn't, and maybe it's context dependent. Just raising awareness.